test.js

```diff
@@ -121,6 +121,20 @@ test('.stopAndPersist() - with new symbol and text', macro, spinner => {
 	spinner.stopAndPersist({symbol: '@', text: 'all done'});
 }, /@ all done/);
 
+test('spinner with default indent (0)', t => {
+	const spinner = new Ora('foo');
+	t.falsy(spinner.indent);
+});
+
+test('spinner with indent (7)', t => {
```

Comment:
I think we should test if the actual output is indented to 7 spaces. Additionally I would remove the test above (0).. I feel it is unnecessary.